spec/factories/external_resources.rb

```diff
@@ -1,12 +1,7 @@
 FactoryBot.define do
   factory :external_resource do
     name { "MyString" }
-    website_url { "MyString" }
```

Comment:
Remove some of these non-required fields. The scaffolds add all fields by default.